api/httpjson/RPCserver.go

```diff
@@ -59,7 +60,18 @@ func NewServer(localNode *node.LocalNode, wallet *vault.Wallet) *RPCServer {
 // Handle is the funciton that should be called in order to answer an rpc call
 // should be registered like "http.HandleFunc("/", httpjsonrpc.Handle)"
 func (s *RPCServer) Handle(w http.ResponseWriter, r *http.Request) {
+	host, _, err := net.SplitHostPort(r.RemoteAddr)
+	if err == nil {
+		limiter := ratelimiter.GetLimiter("rpc:"+host, config.Parameters.RPCIPRateLimit, int(config.Parameters.RPCIPRateBurst))
+		if !limiter.Allow() {
+			log.Infof("RPC connection limit of %s reached", host)
+			w.WriteHeader(http.StatusTooManyRequests)
+			return
+		}
+	}
+
 	if !s.limiter.Allow() {
+		log.Infof("RPC connection limit reached", host)
```

Comment:
`host` havn't added to the log string

Reply:
Fixed, no need to log host because this is overall rate limit